File: app-backend/src/main/java/rs/ac/singidunum/appbackend/models/OrderModel.java
package rs.ac.singidunum.appbackend.models;

import lombok.Data;

import java.time.LocalDate;
import java.util.List;

@Data
public class OrderModel {
    private String id;
    private List<OrderItem> items;
    private UserModel userModel;
    private double totalPrice;
    private LocalDate dateCreated;
    private String status; // zavrsena, tekuca, otkazana
//    private int score; // samo ako je porudzbina zavrsena - kako ovo prikazati/resiti, da li se ovde zapravo cuva
}

File: BIS-project/src/app/models/orderModel.ts
import { OrderItemModel } from "./orderItemModel";
import { UserModel } from "./userModel";

export class OrderModel{
  id?: string;
  items: OrderItemModel[];
  user: UserModel;
  totalPrice: number;
  dateCreated?: Date;
  status: string;
  paymentOption: string;
}
